### ISSUE TITLE
Mismatch package version between NPM and Github

### ISSUE THREAD

**Issue by kelvien** [airbnb/javascript#1759]
I downloaded the latest version 16.1.0 because I am expecting to use this specific commit: https://github.com/airbnb/javascript/commit/4817a6b75cd44fe8b28a16b06d33a5c65603a74c#diff-3077b2e090cd9baca8d3b203af907422
But I found that I was not able to find this changes reflected on my node_modules. I verified that I have this package on version 16.1.0 by checking the package-lock.json and found:
```json
{
  "requires": true,
  "lockfileVersion": 1,
  "dependencies": {
    "eslint-config-airbnb": {
      "version": "16.1.0",
      "resolved": "https://registry.npmjs.org/eslint-config-airbnb/-/eslint-config-airbnb-16.1.0.tgz",
      "requires": {
        "eslint-config-airbnb-base": "12.1.0"
      }
    },
...
```
I have tried clearing my NPM cache as well `npm cache clean --force`.
Is it possible that NPM is not storing the latest version? Can someone verify this?

**Comment by ljharb** [comment]
That commit isn’t yet released (you can tell by seeing no version tags on the page for the commit). When the next version of the configs are released, it will be included (no timeframe atm)

**Comment by kelvien** [comment]
Thanks @ljharb 
I'm not sure if I'm understanding the release process. I assume that changes will be available on version 16.1.0, but won't this break any automated process if the code is eventually released and happen to generate new linting errors to any existing code?

**Comment by ljharb** [comment]
The next release of both configs will be semver-major.

In general, things that change the guide are major, things that create new warnings but only if you disobeyed the guide in the first place are minor, and everything else is patch.
